tests/DatabaseHandlerTest.php

```diff
@@ -148,6 +151,33 @@ public function testPushAndPopWithPriority(): void
         $this->assertSame($payload, $result->payload);
     }
 
+    /**
+     * @throws ReflectionException
+     */
+    public function testPushWithDelay(): void
```

Comment:
Can you add an assert that between time now and time the queue is available_at would be more or less equal to 60 seconds?

Reply:
Added. If this is not what you had in mind, please add a code suggestion.

Reply:
Thanks